### PR TITLE
WIP - Update Sourcegraph OSS features

### PR DESCRIPTION
**Pull request by abschu**
- Clarified what is/isn't included in Sourcegraph OSS
- Added guidance to determine when new features should be added to Sourcegraph OSS
- Updated plan names for paid plans.

### CODE REVIEW

**Review by abschu — COMMENTED** [review]
Updated

**Review by abschu — COMMENTED** [review]
Clarified access controls

**Review by abschu — COMMENTED** [review]
Add example for search aggregations.

**Review by ryphil — APPROVED** [review]

File: content/departments/engineering/product/process/gtm/licensing.md [hunk 1]
```diff
@@ -33,19 +35,35 @@ Benefits to the OSS option include:
 
 If someone is talking to Sales or CE and says they’re using the OSS version, they are 99% of the time using the free tier of Sourcegraph Enterprise. If they say they’re hitting a user limit, they’re using the free version of Sourcegraph Enterprise, not Sourcegraph OSS.
 
+### Determining which new features are added to Sourcegraph OSS
+
+Sourcegraph OSS only includes core search functionality. We will only add new features to Sourcegraph OSS if both of the following are true:
+
+1. They are required for parity with open-source code search tools, and
```
Comment (jdorfman) [inline content/departments/engineering/product/process/gtm/licensing.md]: Our style guide suggests "open-source" should be "open source"

File: content/departments/engineering/product/process/gtm/licensing.md [hunk 2]
```diff
@@ -33,19 +35,35 @@ Benefits to the OSS option include:
...
+2. They don’t significantly reduce the differentiation of Sourcegraph Enterprise or Sourcegraph as a code intelligence platform.
+
+If you believe a feature you are working on should be added to Sourcegraph OSS or you aren't sure if your feature should be added to Sourcegraph OSS, please send a note to #pricing and the team will either approve the addition or help determine if it should be added.
+
+Example: Search aggregations powered by Code Insights
+
+1. No other open-source code search tools provide this functionality, and
```
Comment (jdorfman) [inline content/departments/engineering/product/process/gtm/licensing.md]: "open source"

**Review by jdorfman — CHANGES_REQUESTED** [review]
2 small typos, everything else looks 👍

**Review by jdorfman — APPROVED** [review]

**Review by abschu — COMMENTED** [review]
Changed open-source to open source

**Pull request merged**